packages/mockyeah-docs/src/pages/Contributing.md

```diff
@@ -17,6 +17,28 @@ $ cd mockyeah
 $ nvm install v6
 $ nvm use
 
+# install dependencies and link monorepo modules
+npm install
+npm start
+
 # if tests pass, you're good to go
 $ npm test
 ```
+
+Open the [docs site at http://127.0.0.1:8000](http://127.0.0.1:8000).
+
+## Troubleshooting
+
+Mockyeah is a monorepo project, the dev tool `lerna` is required to connect the sub-modules in the `./packages` directory. If `lerna` doesn't run when you install the dependencies you can run it manually.
+
+```sh
+# if lerna is installed globally
+lerna bootstrap
+
+# or run it locally
+./node_modules/.bin/lerna bootstrap
```

Comment:
Should we create an `npm run lerna` script to make this easier? Or suggest people use `npx lerna`?

Reply:
🤔 Hmmm... There's so many ways to do it. What do you recommend?

Reply:
Let's do `npx lerna` since that's easiest and supported since `npm@5.2.0` so it should be available to all users with our minimum Node version `8.16.0` per https://nodejs.org/en/download/releases/.